src/app/shell/personal-cabinet/provider/create-workshop/create-teacher/teacher-form/teacher-form.component.ts

```diff
@@ -11,10 +12,16 @@ export class TeacherFormComponent implements OnInit {
 
   @Input() index: number;
   @Input() TeacherFormGroup: FormGroup;
+  descriptionMaxLength = 300;
```

Comment:
move to constants folder as for workshop description
![image](https://user-images.githubusercontent.com/27493959/123821495-1d52e300-d904-11eb-9a05-a8248784265c.png)